google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -60,6 +60,7 @@ interfaces:
       - topic
     retry_codes_name: idempotent
     retry_params_name: default
+    timeout_millis: 30000
```

Comment:
You're right, we shouldn't ever have `timeout_millis` < `initial_rpc_timeout_millis`. https://github.com/googleapis/googleapis/issues/54

Note that this change just preserved the current behavior for the Python clients and AFAIK is ignored by other languages at the moment.